play-ws-standalone/src/main/scala/play/api/libs/ws/WSClientConfig.scala

```diff
@@ -29,3 +30,24 @@ case class WSClientConfig(
   compressionEnabled: Boolean = false,
   ssl: SSLConfigSettings = SSLConfigSettings())
 
+object WSClientConfig {
+
+  /**
+   * Creates WSClientConfig from the default application configuration.
+   *
+   * @return a WSClientConfig configuration object.
+   */
+  def forConfig(): WSClientConfig =
```

Comment:
for? `from`? Confusing name

Reply:
I agree, but this is how it was done while creating AHC config. Therefore I decided to keep the consistency.

Reply:
i see ok